samples/Azure.Analytics.Purview.Account/Generated/PurviewAccountsClient.cs

```diff
@@ -1123,9 +1123,10 @@ IEnumerable<Page<BinaryData>> CreateEnumerable(string nextLink, int? pageSizeHin
         /// <summary> Initializes a new instance of PurviewAccountCollections. </summary>
         /// <param name="collectionName"> The String to use. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="collectionName"/> is null. </exception>
```

Comment:
Could we replace "is null" with "is <code>null</code>"?
See: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Blobs.Batch/src/Shared/StringSegment.cs#L43

(Note: if it is not a small change, then let's not worry about it.)

Reply:
Done.

Reply:
Sorry, had to revert it for now. Change footprint is huge. Let's make it a separate PR.